homeassistant/components/apcupsd/config_flow.py

```diff
@@ -46,11 +46,7 @@ async def async_step_user(
             return self.async_show_form(step_id="user", data_schema=_SCHEMA)
 
         host, port = user_input[CONF_HOST], user_input[CONF_PORT]
-
-        # Abort if an entry with same host and port is present.
         self._async_abort_entries_match({CONF_HOST: host, CONF_PORT: port})
-
-        # Test the connection to the host and get the current status for serial number.
```

Comment:
The code here is pretty self-explanatory so I removed these comments for simplicity.